### PR TITLE
chore: Add OG data

### PR DESCRIPTION
**Pull request by ffflorian**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
     />
     <meta name="theme-color" content="#000000" />
     <meta name="google-site-verification" content="eq6QL__YM9oSWEVE3sms7QBeUaxLmwY69ZPM5YWQZ-c" />
+    <meta name="description" content="Sort JSON objects" />
```
Comment (lipis) [inline public/index.html]:
```suggestion
    <meta name="description" content="Sort JSON objects online" />
```

**Comment by ffflorian** [comment]
> Fix some warnings https://developers.facebook.com/tools/debug/sharing/?q=https%3A%2F%2Fsortjson.com

screenshot from 2019-02-19 13-43-24

Nope :smiley:

**Comment by ffflorian** [comment]
> Should also pass this one: https://cards-dev.twitter.com/validator

screenshot from 2019-02-19 13-43-57

Also nope

**Comment by lipis** [comment]
Well, even if you don't agree with some of their policies, being able to share this tool on these platforms is beneficial. Besides the fact that as web developers we should be able to test these pages. For the same reasons we try to deliver the same experience on all OS platforms and as many browsers as possible even if we are not Safari or MS Edge users :)

**Comment by ffflorian** [comment]
> Well, even if you don't agree with some of their policies, being able to share this tool on these platforms is beneficial. Besides the fact that as web developers we should be able to test these pages. For the same reasons we try to deliver the same experience on all OS platforms and as many browsers as possible even if we are not Safari or MS Edge users :)

That's true, but I just don't want to create an account only for using their validators :slightly_smiling_face: 

I'll use https://search.google.com/structured-data/testing-tool for now

**Comment by lipis** [comment]
For the same reasons that we have different devices for testing (Macbook, PC, Tablets, Phones) I think it makes sense to have testing accounts in these platforms.

**Comment by ffflorian** [comment]
> For the same reasons that we have different devices for testing (Macbook, PC, Tablets, Phones) I think it makes sense to have testing accounts in these platforms.

Good idea!

**Comment by lipis** [comment]
https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fsortjson.com 85/100

**Pull request merged**